package.json

```diff
@@ -135,7 +138,9 @@
     "dev:webui": "babel-node tools/dev.server.js",
     "build:webui": "npm run pre:webpack && webpack --config tools/webpack.prod.config.babel.js",
     "build:docker": "docker build -t verdaccio . --no-cache",
-    "build:docker:rpi": "docker build -f Dockerfile.rpi -t verdaccio:rpi ."
+    "build:docker:rpi": "docker build -f Dockerfile.rpi -t verdaccio:rpi .",
+    "prettier-list":"./node_modules/.bin/prettier -l --config .prettierrc.json './{test,src,tools}/**/*.js'",
```

Comment:
`./node_modules/.bin/` is not need it. `npm` map `prettier` from that location anyway

Reply:
Okay. I'll update this.